data/templates/legacy-article.mst

```diff
@@ -11,13 +11,12 @@
     {{/title}}
     <section id="inside-content" class="section">
         <div>{{{body-html}}}</div>
-        <hr>
```

Comment:
I noticed you removed this, but also that you moved the disclaimer out of this section, wouldn't that be a problem considering that the selectors [here](https://github.com/endlessm/eos-knowledge-lib/pull/1267/commits/94249c5614659360e65fe4e47e1996337759d42b#diff-db16cecbc2ecc49d958cd74c4b659d68R526) assume that this going inside that section?

Please let me know if I am missing something :)